solidity/test/bridge/Bridge.Deployment.test.ts

```diff
@@ -17,12 +18,15 @@ describe("Bridge - Deployment", async () => {
   let esdm: SignerWithAddress
 
   let bridge: Bridge
+  let bridgeGovernance: BridgeGovernance
   let bridgeProxy: TransparentUpgradeableProxy
   let proxyAdmin: ProxyAdmin
 
   before(async () => {
     await deployments.fixture()
     ;({ deployer, governance, esdm } = await helpers.signers.getNamedSigners())
+    // eslint-disable-next-line @typescript-eslint/no-extra-semi
+    bridgeGovernance = await helpers.contracts.getContract("BridgeGovernance")
```

Comment:
```suggestion

    bridgeGovernance = await helpers.contracts.getContract("BridgeGovernance")
```

Reply:
part of [40d1f5f](https://github.com/keep-network/tbtc-v2/pull/329/commits/40d1f5f0583534c0683a5f0b871608c216b85900)